docs/reference/ingest.asciidoc

```diff
@@ -448,11 +448,11 @@ configuration. See {fleet-guide}/install-standalone-elastic-agent.html[Install s
 
 [discrete]
 [[pipelines-in-enterprise-search]]
-=== Pipelines in Enterprise Search
+=== Pipelines for Search indices
```

Comment:
Do we want to capitalize "Search" as we move from "Enterprise Search" to "Search"? Maybe lowercase: "search indices" and "search use cases" would be better?

Reply:
There is a "Search" section of Kibana, and I was trying to attach this feature to that section, but I think you're right. This will be more trouble than it's worth, and it'd be better to just lowercase it. I'll update, thanks!

Reply:
**++**